metadata-io/src/main/java/com/linkedin/metadata/search/ranker/SimpleRanker.java

```diff
@@ -28,7 +28,7 @@ public SimpleRanker() {
   @Override
   public Pair<Double, Double> score(SearchEntity searchEntity) {
     Features features = Features.from(searchEntity.getFeatures());
-    return Pair.of(-features.getNumericFeature(Features.Name.RANK_WITHIN_TYPE, 0.0),
-        features.getNumericFeature(Features.Name.NUM_ENTITIES_PER_TYPE, 0.0));
+    return Pair.of(Optional.ofNullable(searchEntity.getScore()).orElse(0.0),
```

Comment:
Do you mind explaining what impact this has? We are now using the search score (if it exists) to rank instead of the number of entities per type? 

Remind me why we have a Pair<Double, Double> here? I don't quite get the data structure

Reply:
Pair is simple. it means that it will rank in order. The first element is used to order first. If the first element is the exact same, it will use the second element. 

I mean here, we may as well just use the score bc the probability of that being exactly the same is very low. 

So overall impact is that  we will stop mixing entity types in an artificial way. Before we would do dataset - chart - dashboard - dataset - chart -dashboard. Now, it will be based on the match score, which from trying out various queries seemed reasonable. I don't think there is a compelling reason to artificially mix in different entity types.

Reply:
makes sense to me - thanks